hydroserving/client/client.py

```diff
@@ -0,0 +1,180 @@
+from enum import Enum
+from typing import Union, Dict, List
+
+import grpc
+import hydro_serving_grpc as hs_grpc
+import hydro_serving_grpc.gateway as hsg
+import pandas as pd
+from google.protobuf import empty_pb2
+from hydro_serving_grpc import manager as hsm
+from loguru import logger
+
+from .contract import HSContract, ContractViolationException
+from .type_conversions import *
+import numpy as np
+
+
+class HSApplication:
```

Comment:
?

Reply:
?

Reply:
Are you serious? Whats the point of this class?